.github/workflows/ubuntu_20.yml

```diff
@@ -100,6 +100,24 @@ jobs:
             -DCPACK_GENERATOR=TGZ \
             -DCMAKE_COMPILE_WARNING_AS_ERROR=ON
 
+  CXX_Unit_Tests:
+    name: C++ unit tests
+    needs: [ Docker, Build, Smart_CI ]
+    uses: ./.github/workflows/job_cxx_unit_tests.yml
+    with:
+      runner: 'aks-linux-4-cores-16gb'
+      image: ${{ fromJSON(needs.docker.outputs.images).ov_test.ubuntu_20_04_x64 }}
+      affected-components: ${{ needs.smart_ci.outputs.affected_components }}
+
+  CPU_Functional_Tests:
```

Comment:
I suppose it's too much to run CPU tests on each Ubuntu version.
I think we need to run on single selected one

Reply:
Agreed. I did miss that the functional tests are running only on one Ubuntu in Jenkins; I'll remove them from all but U22.